### PR TITLE
fix(ui5-popover): fix content truncation

### PR DESCRIPTION
**Pull request by TeodorTaushanov**
#4259

### CODE REVIEW

**Review by LidiyaGeorgieva — APPROVED** [review]

File: packages/main/src/themes/Dialog.css [hunk 1]
```diff
@@ -1,5 +1,4 @@
 :host {
-	min-width: 20rem;
```
Comment (vladitasev) [inline packages/main/src/themes/Dialog.css:2]: why were these removed?

Comment (TeodorTaushanov) [reply packages/main/src/themes/Dialog.css]: Removed by mistake.

**Review by LidiyaGeorgieva — APPROVED** [review]

**Comment by vladitasev** [comment]
@MapTo0 Marti, the deleted code is introduced by a PR of yours, fixing this issue: https://github.com/SAP/ui5-webcomponents/issues/853 

Please have a look

File: packages/main/src/themes/PopupsCommon.css [hunk 2]
```diff
@@ -1,7 +1,6 @@
 :host {
     display: none;
     position: fixed;
-    min-width: 6.25rem;
```
Comment (vladitasev) [inline packages/main/src/themes/PopupsCommon.css:4]: I saw that you reverted the removed `min-width: 20px` for Dialog.js, however this one is still deleted. I checked and Popover.css does not specify min-width by itself. Therefore `PopupsCommon.js` is responsible for setting it. After this change, popovers will no longer have a minimum width, which might be interpreted as a breaking change. Is there a reason to remove it?

Comment (TeodorTaushanov) [reply packages/main/src/themes/PopupsCommon.css:4]: If min-width is set to the "host element" and there is not enough space, the Popover looks broken:
MinWidth

If it is removed, the Popover looks like:
NoMinWidth

If we want to have some minimum width, we can add it to the "popover-root":
MinWidthInside

which doesn't look so bad and we can use this approach. What do you think?

Comment (vladitasev) [reply packages/main/src/themes/PopupsCommon.css:4]: I think it makes sense

**Comment by ilhan007** [comment]
FYI: We try to start the commit messages with a verb, small letter and in imperative form, for example the current one would become:

fix(ui5-popover): fixed content truncation

When you press the "Squash and merge" button you will have the chance to fix the commit message.
Note: If you edit the PR commit message via the Github UI, which is also fine, you will anyway have to update it once more when you press the "Squash and merge" button.

It's not a big deal, but as the release change log is generated on these commit messages, it will look inconsistent.

File: packages/main/src/themes/PopupsCommon.css (hunk 2 above)
Comment (TeodorTaushanov) [reply packages/main/src/themes/PopupsCommon.css:4]: min width is added to the popover shadow root

**Review by gmkv — APPROVED** [review]
